gc/base/standard/ConfigurationGenerational.cpp

```diff
@@ -219,4 +219,20 @@ MM_ConfigurationGenerational::defaultMemorySpaceAllocated(MM_GCExtensionsBase *e
 	extensions->setGuaranteedNurseryRange(nurseryStart, (void*)UDATA_MAX);
 }
 
+uintptr_t
+MM_ConfigurationGenerational::calculateDefaultRegionSize(MM_EnvironmentBase *env)
+{
+	uintptr_t regionSize = STANDARD_REGION_SIZE_BYTES;
+
+#if defined(OMR_GC_CONCURRENT_SCAVENGER)

```

Comment:
Well, What I should do? Use compile flag OMR_GC_CONCURRENT_SCAVENGER + new Aleks API? Otherwise to have run-time check only I should move CS specific variables out from OMR_GC_CONCURRENT_SCAVENGER flag to make them common. Is it what we need?